### DISCUSSION TITLE
Install Help

### DISCUSSION THREAD

**Discussion by julianflack** [publicsamples/Oi-Grandad#23] [Q&A] [answered]
I downloaded the AU version through the dmg and it said the software was installed successfully, but I don't know where the necessary files are now or what to do with them. How can I get this working?

**Comment by RickTaylor79** [comment, marked as answer]
Open your DAW or plugin host and Oi, Grandad! will be available as a plugin synthesiser (refer to your DAW documentation for specifics).

If you want to install the presets and extra content, then download and extract [this zip ](https://github.com/publicsamples/Oi-Grandad/releases/download/1.1.3/AdditionalContent.zip) then:

1. Open Finder
2. Select "Go" then "Go to folder"
3. Type in /Users/YOURNAME/Library/Application Support/ (replacing YOURNAME with the login name for your Mac)
4. Move the modularsamples folder to Application Support 

Now when you open Oi, Grandad! you can load the presets by pressing the presets button.

Check out the tutorial videos to learn the basics of the plugin:

https://youtube.com/playlist?list=PL8Rp79UJ5uBC-SntUW93i3953yPq0DeSF

**Reply by julianflack** [reply]
Oops, I forgot Avid is a racket and Pro Tools can't do AU plugins. Would love an AAX version if possible in the future!

**Reply by RickTaylor79** [reply]
Sorry, but I've no interest in developing this for AAX/pro tools. I'd have get a pro tools license to test it, jump through hoops to get the SDK. Life's too short :)

That said, Oi, Grandad is open source, and built with a reasonably easy to use, GUI framework  [HISE](https://github.com/christophhart/HISE). Why not build your own AAX version? It should be as simple as:

1. Install HISE (I think most recent [binary](https://github.com/christophhart/HISE/releases/tag/3.0.1) should work)
2. Clone and repository
3. Obtain the AAX SDK
4. Export AAX
5. Test
